Cargo.toml

```diff
@@ -50,3 +51,10 @@ defmt = [
     "embedded-tls?/defmt",
     "nourl/defmt",
 ]
+
+# For esp32, re-export those features as required by Cargo.
+# This will also automatically enable `esp-mbedtls`.
+esp32 = ["esp-mbedtls/esp32"]
+esp32c3 = ["esp-mbedtls/esp32c3"]
+esp32s2 = ["esp-mbedtls/esp32s2"]
+esp32s3 = ["esp-mbedtls/esp32s3"]
```

Comment:
They don't really need to be reexported do they? You can configure them directly on your app and features will be merged.

In any case I think they should be prefixed with esp-mbedtls, since you could still use embedded-tls on esp32.

Reply:
I tried that originally, but it seems like Cargo doesn't allow specifying features for transitive dependencies. Unfortunately, we have to re-export them.

Reply:
You can add a dependency on esp-mbedtls = { version ... features = ["esp32"]}  in your application, which should also enable the feature for the transitive dependency.